src/util/alloc/immix_allocator.rs

```diff
@@ -28,9 +26,9 @@ pub struct ImmixAllocator<VM: VMBinding> {
     /// Is this a copy allocator?
     copy: bool,
     /// Bump pointer for large objects
-    large_cursor: Address,
+    pub(crate) large_cursor: Address,
```

Comment:
You could coalesce the `large_{cursor,limit}` fields into one `large_bump_pointer` as well.